### DISCUSSION TITLE
How to join a clan?

### DISCUSSION THREAD

**Discussion by RyanGitHub101** [codewars/codewars.com#2863] [Q&A]
<img width="1112" alt="Screenshot 2022-11-14 at 16 55 57" src="https://user-images.githubusercontent.com/96314042/201719397-94e3cde6-d320-4f58-b100-8389ab547057.png">

Hello, guys I need help on how to join a clan.

**Comment by Madjosz** [comment]
You go to Account Settings, type in the name of a clan and save, that's all.
Please note that clans are mostly useless. You just mutually follow each clan member and can see them on your dashboard.
See also: https://docs.codewars.com/community/following#clans
